installs/install-pip.sh

```diff
@@ -25,7 +25,7 @@ set -e
 set -o pipefail
 
 "${LOCAL}/help/assert-tool.sh" python3 --version
-"${LOCAL}/help/assert-tool.sh" pip --version
+"${LOCAL}/help/assert-tool.sh" pip3 --version
 
 "${LOCAL}/help/sudo.sh" python3 -m pip install --upgrade pip
```

Comment:
@timur-harin but here it's `pip` instead of `pip3`? What is right?

Reply:
Yes, because inside python3 only pip exists 

`
timur ~  pip --version
zsh: command not found: pip
timur  ~  pip3 --version
pip 24.0 from /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pip (python 3.11)
timur  ~  python3 -m pip --version
pip 24.0 from /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pip (python 3.11)
timur  ~  python3 -m pip3 --version
/usr/local/bin/python3: No module named pip3
`